origin/blobserver/server.go

```diff
@@ -385,45 +433,68 @@ type localReplicationHook struct {
 }
 
 func (h *localReplicationHook) Run(d core.Digest) {
+	start := time.Now()
+	log.With("digest", d.Hex()).Info("Starting local replication")
 	timer := h.server.stats.Timer("replicate_blob").Start()
 	if err := h.server.replicateBlobLocally(d); err != nil {
 		// Don't return error here as we only want to cache storage backend errors.
-		log.With("blob", d.Hex()).Errorf("Error replicating remote blob: %s", err)
+		duration := time.Since(start)
+		log.With("digest", d.Hex(), "duration_ms", duration.Milliseconds()).Errorf("Error replicating remote blob: %s", err)
```

Comment:
Should we log in seconds instead ? In Production we have seen blobs being replicated for ~ 10 minutes which is ~ 600,000 ms. I believe that might make some of the logs difficult to understand.